main.c

```diff
@@ -122,8 +122,8 @@
 uint8_t value_out = 0;
 #endif
 
-#if MICROPY_ENABLE_PYSTACK
-static size_t PLACE_IN_DTCM_BSS(_pystack[CIRCUITPY_PYSTACK_SIZE / sizeof(size_t)]);
+#if MICROPY_ENABLE_PYSTACK && CIRCUITPY_OS_GETENV
```

Comment:
With "getenv" being used so far down in the stack, I think we should acknowledge that os_getenv is a mandatory module now.

Reply:
Personally, I feel like super-slim builds still have their place.
getenv isn't needed in those.